joplin/base/signals.py

```diff
@@ -50,15 +50,9 @@ def create_build_aws(instance, publish=False, request=None):
     logger.debug("create_build_aws() Starting task")
 
     slack_message = ""
-    publish_action = ""
-
-    if (publish):
-        publish_action = "published"
-    else:
-        publish_action = "unpublished"
```

Comment:
Was this unused code?

Reply:
this used to be used for our slack message (either say we published or unpublished a page). I changed it so we just pass in a publish_action as a string for publish/unpublish, and have the default just be "edited"